src/Microsoft.Diagnostics.Runtime/src/Common/ClrType.cs

```diff
@@ -54,9 +54,9 @@ public abstract class ClrType :
         public virtual ulong LoaderAllocatorHandle => 0;
 
         /// <summary>
-        /// Gets the address of the <c>System.Runtime.Loader.AssemblyLoadContext</c> object.
+        /// Gets the address of the <c>AssemblyLoadContext</c> object.
         /// </summary>
-        public virtual ulong AssemblyLoadContextHandle => 0;
+        public virtual ulong AssemblyLoadContextAddress => 0;
```

Comment:
This is a breaking change.  The original version of this PR had the old `AssemblyLoadContextHandle` marked [Obsolete], which is the right thing to do here.  Otherwise this PR will have to wait until 2.1.

```suggestion
        public virtual ulong AssemblyLoadContextAddress => 0;
        
        [Obsolete("Use AssemblyLoadContextAddress.")]
        public virtual ulong AssemblyLoadContextHandle =>AssemblyLoadContextAddress;
```

Reply:
This API was added very lately and hasn't been published, I think.